Different Sparkassen do not work without force_mt940

Hi, first thanks for the awesome work.

I started with the fints-importer just now.
I use the latest docker image from docker hub.

When using Sparkasse Sparkasse Schweinfurt-Haßberge or Sparkasse Coburg-Lichtenfels without force_mt940 I get that error in the browser:
Fatal error: Uncaught Error: Typed property Fhp\Model\StatementOfAccount\Transaction::$name must not be accessed before initialization in /vendor/nemiah/php-fints/lib/Fhp/Model/StatementOfAccount/Transaction.php:317 Stack trace: #0 /vendor/twig/twig/src/Extension/CoreExtension.php(1913): Fhp\Model\StatementOfAccount\Transaction->getName() #1 /vendor/twig/twig/src/Environment.php(420) : eval()'d code(132): Twig\Extension\CoreExtension::getAttribute(Object(Twig\Environment), Object(Twig\Source), Object(Fhp\Model\StatementOfAccount\Transaction), 'getName', Array, 'method', false, false, false, 37) #2 /vendor/twig/twig/src/Template.php(446): __TwigTemplate_5f875ed6b23e69a21cb1859b3388952c->block_content(Array, Array) #3 /vendor/twig/twig/src/Environment.php(420) : eval()'d code(58): Twig\Template->yieldBlock('content', Array, Array) #4 /vendor/twig/twig/src/Template.php(402): __TwigTemplate_af2f401eda7be36f9c06e4eac177dfd7->doDisplay(Array, Array) #5 /vendor/twig/twig/src/Environment.php(420) : eval()'d code(51): Twig\Template->yield(Array, Array) #6 /vendor/twig/twig/src/Template.php(402): __TwigTemplate_5f875ed6b23e69a21cb1859b3388952c->doDisplay(Array, Array) #7 /vendor/twig/twig/src/Template.php(358): Twig\Template->yield(Array, Array) #8 /vendor/twig/twig/src/Template.php(373): Twig\Template->display(Array) #9 /vendor/twig/twig/src/TemplateWrapper.php(51): Twig\Template->render(Array) #10 /vendor/twig/twig/src/Environment.php(333): Twig\TemplateWrapper->render(Array) #11 /app/GetImportData.php(108): Twig\Environment->render('show-transactio...', Array) #12 /app/index.php(76): App\StepFunction\GetImportData() #13 {main} Next Twig\Error\RuntimeError: An exception has been thrown during the rendering of a template ("Typed property Fhp\Model\StatementOfAccount\Transaction::$name must not be accessed before initialization") in "show-transactions.twig" at line 37. in /app/public/html/show-transactions.twig:37 Stack trace: #0 /vendor/twig/twig/src/Environment.php(420) : eval()'d code(58): Twig\Template->yieldBlock('content', Array, Array) #1 /vendor/twig/twig/src/Template.php(402): __TwigTemplate_af2f401eda7be36f9c06e4eac177dfd7->doDisplay(Array, Array) #2 /vendor/twig/twig/src/Environment.php(420) : eval()'d code(51): Twig\Template->yield(Array, Array) #3 /vendor/twig/twig/src/Template.php(402): __TwigTemplate_5f875ed6b23e69a21cb1859b3388952c->doDisplay(Array, Array) #4 /vendor/twig/twig/src/Template.php(358): Twig\Template->yield(Array, Array) #5 /vendor/twig/twig/src/Template.php(373): Twig\Template->display(Array) #6 /vendor/twig/twig/src/TemplateWrapper.php(51): Twig\Template->render(Array) #7 /vendor/twig/twig/src/Environment.php(333): Twig\TemplateWrapper->render(Array) #8 /app/GetImportData.php(108): Twig\Environment->render('show-transactio...', Array) #9 /app/index.php(76): App\StepFunction\GetImportData() #10 {main} thrown in /app/public/html/show-transactions.twig on line 37

In logfile with trace on it looks good.
[Wed Mar 18 11:10:33 2026] [TRACE] CAMT XML array count: 447
[Wed Mar 18 11:10:33 2026] [TRACE] First CAMT XML length: 3084

Also I see the transactions in the xml file log.
Can I help you to get this fixed? 

Another short question: is it normal that ING still uses mt940 (without force?)

I have the same problem with ING. With `force_mt940=true` it is working. Otherwise it does not find any transaction.
But I have this problem only with my "Girokonto". I also have two "tagesgeldkonto" which are working flawlessly.

This is the log
```
[Fri Apr 24 07:49:28 2026] [INFO] Using statement format: camt
[Fri Apr 24 07:49:29 2026] [INFO] Parsing CAMT XML format
[Fri Apr 24 07:49:29 2026] [ERROR] CAMT XML parsing failed: Typed property Fhp\Model\StatementOfAccount\Transaction::$structuredDescription must not be accessed before initialization
[Fri Apr 24 07:49:29 2026] [INFO] No transactions found for date range: 2026-03-30 to 2026-04-24
```

--- a/app/StatementOfAccountHelper.php
+++ b/app/StatementOfAccountHelper.php
@@ -119,7 +119,11 @@ class StatementOfAccountHelper
                             $name = $partyType->getName();
                             if ($name) {
                                 $transaction->setName($name);
+                            } else {
+                                $transaction->setName("");
                             }
+                        } else {
+                            $transaction->setName("");
                         }
 
                         // Handle single-party transactions where bank only provides one party (yourself)

